packages/forms-web-app/src/validators/householder-planning/eligibility/enforcement-notice-householder.js

```diff
@@ -0,0 +1,17 @@
+const { body } = require('express-validator');
+
+const validEnforcementNoticeHouseholderOptions = ['yes', 'no'];
+
+const ruleEnforcementNoticeHouseholder = () =>
+  body('enforcement-notice')
```

Comment:
Should be enforcement-notice-householder

Reply:
The control is still set as _enforcement-notice_. Are we renaming them as well?